### PR TITLE
Update `upgrade` instructions w/ a few more details

### PR DESCRIPTION
**Pull request by foot**

### CODE REVIEW

File: docs/cluster-management/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+hide_title: true
+---
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="enterprise" />
+</h1>
+
+## Creating your first CAPD Cluster
+
+If you've followed the [Upgrade guide](../enterprise/upgrading.mdx) you should have a management cluster ready to roll. Nex up we'll
```
Comment (dlopro) [inline docs/cluster-management/getting-started.mdx:13]: next*

**Review by dlopro — APPROVED** [review]

File: docs/cluster-management/getting-started.mdx (hunk 1 above)
Comment (foot) [inline docs/cluster-management/getting-started.mdx:13]:
```suggestion
If you've followed the [Upgrade guide](../enterprise/upgrading.mdx) you should have a management cluster ready to roll. Next up we'll
```

Comment (foot) [reply docs/cluster-management/getting-started.mdx]: 🙏

**Comment by steveww** [comment]
To keep the UI safe I would bind 30080 to 127.0.0.1 in the kind config.

**Comment by foot** [comment]
> To keep the UI safe I would bind 30080 to 127.0.0.1 in the kind config.

Good one, I'll do another pass on this tomorrow

**Pull request merged**